decidim-admin/app/views/decidim/admin/scopes/search_results.json.erb

```diff
@@ -0,0 +1,9 @@
+<%
```

Comment:
My original solution used the `to_json` method, but I get stuck with the translatable attribute, so I ended up with this ugly solution. I think I will use ProposalSerializer or CommentSerializer as an example.